src/Datadog.Trace/TracerConstants.cs

```diff
@@ -0,0 +1,7 @@
+namespace Datadog.Trace
+{
+    internal static class TracerConstants
+    {
+        public const string Language = "dotnet";
+    }
+}
```

Comment:
There didn't seem to be any good place to put constants that are .NET Tracer-specific, so I created this internal static class. If there's a better place, let me know.